web-console/tests/e2e/demoAccrual.spec.ts

```diff
@@ -187,17 +188,17 @@ test('Accrual demo test', async ({ page, request }) => {
       'CUSTOMER_CONSUMED_V',
       'CUSTOMER_TOTAL_CREDIT_V',
       'CUSTOMER_BALANCE_V'
-    ]) {
+    ].entries()) {
       await page.getByTestId('button-expand-relations').click()
       await page.getByTestId('box-relation-options').getByTestId(`button-option-relation-${relation}`).click()
-      await page.getByTestId('box-relation-options').waitFor({state: 'hidden'})
-      await expect(page).toHaveScreenshot(`relation ${relation}.png`)
+      await page.getByTestId('box-relation-options').waitFor({ state: 'hidden' })
+      await expect(page).toHaveScreenshot(`5-${i}-relation ${relation}.png`)
     }
   })
 
   await test.step('Stop the pipeline', async () => {
     await page.getByTestId('button-current-pipeline').click()
-    await expect(page).toHaveScreenshot('pipeline details final.png', {
+    await expect(page).toHaveScreenshot('6-1-pipeline details final.png', {
```

Comment:
what are the numbers?

Reply:
Naming convention I introduce to help lookup screenshots faster